frontend/src/lib/utils.tsx

```diff
@@ -724,6 +727,14 @@ export function autocorrectInterval(filters: Partial<FilterType>): string {
     }
 }
 
+export function pluralize(count: number, singular: string, plural?: string, includeNumber: boolean = true): string {
```

Comment:
Maybe some unit tests for this would be in order?

Reply:
Sure!